bin/kalite

```diff
@@ -109,5 +109,9 @@ p = subprocess.Popen(
     env=env,
 )
 
-p.communicate()
+try:
+    p.communicate()
+except KeyboardInterrupt:
+    # We are not going to show this un-interesting trackeback!
```

Comment:
Oh that was there on purpose... Just testing your review eyes aren't sleepy :) :)